homeassistant/components/edl21/sensor.py

```diff
@@ -290,7 +290,7 @@ async def async_setup_platform(
         hass,
         DOMAIN,
         "deprecated_yaml",
-        breaks_in_ha_version="2023.2.0",
+        breaks_in_ha_version="2023.5.0",
```

Comment:
https://github.com/home-assistant/core/pull/87655 was merged on 1st March, so it was included in 2023.4
I suggest to amend this on 2023.6 instead.
```suggestion
        breaks_in_ha_version="2023.6.0",
```